onboarding/source/amb_light_service.c

```diff
@@ -15,25 +22,101 @@
  */
 static void lightServiceTask(void * pvParameters);
 
+
+void adcStartConversion_selChn(adcBASE_t *adc, unsigned channel, unsigned fifo_size, unsigned group)
```

Comment:
Instead of `unsigned`, use `unsigned int` or the appropriate fixed-size integer types (like `uint8_t`)

Reply:
This applies to the rest of your PR as well.